login-workflow/docs/custom-account-details.md

```diff
@@ -0,0 +1,70 @@
+# Custom Account Details
+The `@pxblue/angular-auth-workflow` allows for custom account detail fields to be added during a user's self-registration.   To add custom form fields to the self registration pages, follow the steps below:
+
+### Create a custom `<pxb-create-account>` component.
+
+```angular2
+<ng-template #createAccountPage>
+    <pxb-create-account [accountDetails]="accountDetails"></pxb-create-account>
+</ng-template>
+
+<pxb-auth [createAccountRef]="createAccountPage"></pxb-auth>
+``` 
+
+### Add your custom form content as a `<ng-template>`
+```angular2
+<ng-template #customFormRef>
+  <form>
+      <mat-form-field appearance="fill">
+          <mat-label>Custom Field</mat-label>
+          <input matInput [formControl]="customFormControl" required />
+          <mat-error *ngIf="customFormControl.hasError('required')">
+              Emergency Contact is <strong>required</strong>
+          </mat-error>
+      </mat-form-field>
+  </form>
+</ng-template>
+```
+
+### Populate the `accountDetails` @Input
+
+Each `accountDetails` object has 3 properties:
+
+| Property            | Description                                            |
+| ------------------- | ------------------------------------------------------ | 
+| form                | What to render                                         |
+| formControls        | the login screen                                       |
```

Comment:
Is this right?

Reply:
should it say something like 'a map listing all of the fields in the form'

Reply:
Good catch, this must have been copied/pasted over incorrectly.